docs/openapi/oad/ephemeral.oad.yaml

```diff
@@ -0,0 +1,80 @@
+openapi: 3.1.0
+info:
+  title: Ephemeral Service API
+  description: |
+    This API allows to interact with the Ephemeral service. It provides a single endpoint used to trigger computations.
+  version: 0.1.0
+  license:
+    name: Apache-2.0
+    url: https://github.com/carbynestack/ephemeral/blob/7ce9f830de3c5852afdc6c3c6c591fc5d04a50f7/LICENSE
+  contact:
+    name: Carbyne Stack
+    url: https://carbynestack.io/
+paths:
+  /:
+    post:
+      summary: Run a computation
+      description: |
+        Start a disctributed computation. The result of the computation is stored as a new Amphora Secret referenced
```

Comment:
I would rather say: Trigger execution of an MPC function.

Reply:
Done. Also replaced `program` with `function` in general.